core/src/main/resources/db/migration/V1__initial_schema.sql

```diff
@@ -1,3 +1,5 @@
+ALTER DATABASE CHARACTER SET = utf8;
+
```

Comment:
Alternatives:
1. add to the last migration step the `ALTER DATABASE CHARACTER SET = utf8;`
2. add a new migration step consisting solely of the `ALTER DATABASE` above.